include/openthread/backbone_router.h

```diff
@@ -75,6 +75,16 @@ typedef struct otBackboneRouterConfig
  */
 otError otBackboneRouterGetPrimary(otInstance *aInstance, otBackboneRouterConfig *aConfig);
 
+/**
+ * This function gets the Backbone TMF UDP port in the Thread Network.
+ *
+ * @param[in]   aInstance            A pointer to an OpenThread instance.
+ *
+ * @returns The UDP port of Backbone TMF.
+ *
+ */
+uint16_t otGetBackboneUdpPort(otInstance *aInstance);
```

Comment:
Thanks @zwx1995esp .

I beleive the TMF port should be constant and defined by spec to be `61631`.

We can define it as a constant (instead of an API)?

```c++
#define OT_BACKBONE_ROUTER_TMF_UDP_PORT 61631  ///< The backbone TMF UDP port number.
```

Reply:
Ok, the only requirement this API used is that our ESP port need this port number for the TXT `bb` in the mesh coap mdns. As you said, I think we just define a macro in ESP port, and use this port number for the TXT in mDNS.

Reply:
Sounds good. Thanks. 
BTW, I see you changed the PR to use macro but the PR seems to be closed. Did you intern to close it?

Reply:
Yeah, I close this PR, due to I think this port number is defined by spec, so this port number will be never changed, I just use a hard code port number in our ESP port for the TXT in mDNS.